README.md

```diff
@@ -21,3 +21,11 @@ Do poprawnego wypełnienia bazy danych potrzebne może być jeszcze uruchomienie
 ```bash
 python3 apsi_diploma/manage.py migrate
 ```
+
+Aby zainicjalizować użytkowników, grupy oraz uprawnienia należy uruchomić:
+
+```bash
```

Comment:
This is not a wrokaround, but optional migration used to simplify generating database.

In case you have already initialized database with users, groups etc you would have problems with applying migrations.

Reply:
Alrighty, fair point!